cpu/stm32f1/periph/spi.c

```diff
@@ -210,11 +210,11 @@ int spi_transfer_byte(spi_t dev, char out, char *in)
             return -1;
     }
 
-    while (!(spi->SR & SPI_SR_TXE));
+    while (!(spi->SR & SPI_SR_TXE)) ;
     spi->DR = out;
     transferred++;
 
-    while (!(spi->SR & SPI_SR_RXNE));
+    while (!(spi->SR & SPI_SR_RXNE)) ;
```

Comment:
the introduced space here looks weird to me... Not sure if we have a convention on this?! @OlegHahm: any insights?